server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/burstcoin/wallets/burstwallet/BurstWallet.java

```diff
@@ -111,15 +107,15 @@ public String sendCoins(String destinationAddress, BigDecimal amount, String cry
 
         BigInteger recipientInt = BurstAddressValidator.getAccountIdFromRS(destinationAddress);
         if (recipientInt == null) {
-            log.debug("Invalid destination address");
+            log.debug("Invalid destination address"); // Error?
             return null;
         }
 
         String recipient = recipientInt.toString();
 
         SendResponse res = api.send(masterPassword, accountId, recipient, amount.multiply(NQT).stripTrailingZeros(), DEFAULT_FEE_IN_NXT.multiply(NQT).stripTrailingZeros(), 1440, "sendMoney");
         if (res == null) {
-            log.debug("No response received.");
+            log.debug("No response received."); // Error?
```

Comment:
Should this be `log.error()`?

Reply:
See above, i just  used what had already been done.